content/blog/2020/08/2020-08-12-windows-installers-upgrade.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Windows Installer Upgrades"
+tags:
+- windows
+- platform-sig
+- installers
+author: vsilverman
+---
+
+The purpose of the current guide is to describe the transition from the old Jenkins Windows installer 2.235.2 (32 bit) to the new Jenkins Windows installer 2.235.3 (64 bit)
+
+Let's take a look how Jenkins installation on Windows happened before release of this upgrade.
+
+## Step 1
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/old_installer_1.png[Installer Startup, role=center]
+
+It's evident that branding information is not present here.
+
+## Step 2
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/old_installer_2.png[Installation Directory, role=center]
+
+Not much branding information here as well.
+
+## Step 3
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/old_installer_3.png[Install It, role=center]
+
+The previos installer gave only one option for installing Jenkins - just selecting the destination folder.
+
+## Issues
+
+The current installer has a few issues that the Platform SIG wanted to fix in a new install experience for users.
+
+ 1. The installer only supports 32-bit installations.
+ 2. The user could not select ports or user accounts to run the service on.
+ 3. The installer bundled a 32-bit version of the Java runtime instead of using a pre-existing JRE
+ 4. The installer did not support the experimental support in Jenkins for Java 11
+ 5. The JENKINS_HOME directory was not placed in a good spot for modern Windows
+ 6. There is no branding in the installer.
+ 
+## Road Forward
+
+The experimental Jenkins Windows Installer resolves most of the outlined issues!

```

Comment:
Installer was experimental in Feb 2019.  Is production now.  Has been used in weekly builds since April 2020.
```suggestion
The new Jenkins Windows Installer resolves most of the outlined issues!

```